### ISSUE TITLE
Display Works and Files section to public

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#3455]
### Descriptive summary
The accessibility review team would like to enable the Works and Files section of the work show page to users. They goal is to display individual files/filesets to users so they may download individually.
Two updates to the display need to be made:
- When the user shouldn't have access to any of the options besides `download` the action dropdown should be replaced with a download button
- A new column should be added that indicates if the work is the accessible version or not. POSM is not sure what this should look like just yet, so take your best stab at something. A badge, a check mark, `remediated`. Something along those lines.

### Expected behavior
`Works and Files` section of work show page is visible to unauthed and other public users.
Public users only have the `download` option
A new column exists in the `Works and Files` section that indicates if the FileSet is an accessibility remediated file or not.

**Comment by jsimic** [comment]
Please make the controlled vocabulary for accessibilityFeature more user friendly in display, i.e. "tableOfContents" displays as "Table of Contents"

**Comment by lamtu1** [comment]
@CGillen Got it. I currently have link to 'Show Item Page' for `mask_content` so if they hit the `i understand` button within the UV area, they will unblurred the image from the UV as well as the one from the work & file set. Wondering if this works for that?
